SafeguardSessions/CHANGELOG.md

```diff
@@ -2,6 +2,41 @@
 
 ## [Unreleased]
 
+### Compatible SPS Versions
+
+* 7.3.0, 7.4.0
+
+### Breaking changes
+
+* Power BI Service support has been implemented and the order and type of the input parameters have changed. Before the upgrade, we recommend to check what input parameters were used when connecting to your SPS so that after replacing the old version of the connector with the new version, you know how to fix the order and type of the input parameters. After the upgrade, the input parameters of previously saved .pbix files must be modified according to your observations to work with the new versions of One Identity Safeguard Power BI Connector and One Identity Safeguard Power BI Report Template.

```

Comment:
`Power BI Service support has been implemented and the order and type of the input parameters have changed. Before the upgrade, we recommend that you check what input parameters were used when connecting to your SPS, so that after replacing the old version of the connector with the new version, you know how to fix the order and type of the input parameters. To ensure compatibility with the new versions of the One Identity Safeguard Power BI Connector and One Identity Safeguard Power BI Report Template, we recommend that after the upgrade, you modify the input parameters of previously saved .pbix files according to your observations.`

Reply:
Next PR.